pycromanager/acquire.py

```diff
@@ -482,25 +504,36 @@ def multi_d_acquisition_events(num_time_points=1, time_interval_s=0, z_start=Non
 
     Returns
     -------
-
-    
+    events : dict
     """
-
+    if xy_positions is not None and xyz_positions is not None:
+        raise ValueError("xyz_positions and xy_positions are incompatible arguments that cannot be passed together")
+    order = order.lower()
+    if 'p' in order and 'z' in order and order.index('p') > order.index('z'):
+        raise ValueError("This function requres that the xy position come earlier in the order than z")
+    has_zsteps = z_start is not None and z_step is not None and z_end is not None
+    z_positions = None
+    if xy_positions is not None:
+        xy_positions = np.asarray(xy_positions)
+        z_positions = None
+    elif xyz_positions is not None:
+        xyz_positions = np.asarray(xyz_positions)
+        xy_positions = xyz_positions[:,:2]
+        z_positions = xyz_positions[:,2][:, None]
+
+    if has_zsteps:
+        z_rel = np.arange(z_start, z_end + z_step, z_step)
```

Comment:
This includes one more point than the old version. To me this way make more sense but it is clearly backwards incompatible.. let me know if you'd prefer this to be
```suggestion
        z_rel = np.arange(z_start, z_end, z_step)
```

Reply:
Yeah I think that's fine and probably an upgrade